content/posts/2021/release-0.8.0.md

```diff
@@ -0,0 +1,362 @@
+  scheduling policy will also look at the workflow DAG complexity. If
+  a new worklow C would like to launch 10 jobs at the start, and the
+  workflow D only 2 jobs, and the cluster is busy and the current
+  capacity cannot accept more than 4 jobs at the given time, the
+  workflow D would be preferred over workflow C.
+
+- ``reana_server.environment.REANA_RATELIMIT_GUEST_USER`` and
+  ``reana_server.environment.REANA_RATELIMIT_AUTHENTICATED_USER``
+  allow to set REST API rate limit values.  If your cluster is very
+  busy, you can use values such as "2000 per second" enabling heavy
+  traffic from running a high amount of concurrent workflows.
+
+- ``reana_server.environment.REANA_SCHEDULER_REQUEUE_SLEEP`` allows to
+  set sleep time between processing queued workflows. The default
+  value of 15 seconds may be too conservative, as the scheduler will
+  pause for 15 seconds before looking at the incoming workflow
+  queue. If you are running a lot of concurrent workflows, you may
+  want to lower it to 1 second or even 0 seconds.
+
+- ``reana_workflow_controller.environment.REANA_JOB_STATUS_CONSUMER_PREFETCH_COUNT``
+  allows to tweak the MQ prefetch count for the job status
+  consumer. The default value was tested to behave well up to a
+  thousand of concurrent workflows, so should be probably fine in your
+  setups.
+
+### How to upgrade existing REANA 0.7 clusters
+
+If you are a REANA cluster administrator and you would like to upgrade
+from REANA 0.7.x to REANA 0.8.0, please follow the steps described
+below.
+
+#### 1. Upgrade the cluster using Helm
+
+```console
+$ helm upgrade reana reanahub/reana --version 0.8.0
+```
+
+You may set various Helm values specified above; see also the full
+list of [REANA Helm
+values](https://github.com/reanahub/reana/blob/0.8.0/helm/reana/README.md).
+
+#### 2. Label the cluster nodes
+
+REANA 0.8 release series uses more cluster node labels which allows to
+separate nodes dedicated to running infrastructure pods, runtime user
+workflow batch pods, runtime user job pods, and runtime user
+interactive sessions. Setting the new values allows to prevent
+exceptional situations with user jobs, such as exhausting node memory,
+from affecting other pods.
+
+We therefore recommend to upgrade your cluster node labels. For
+example, if you have four nodes in the cluster, you can label them as
+follows:
+
+```console
+$ kubectl label node reana-cluster-node-0 reana.io/system=infrastructure
+$ kubectl label node reana-cluster-node-1 reana.io/system=runtimebatch
+$ kubectl label node reana-cluster-node-2 reana.io/system=runtimejobs
+$ kubectl label node reana-cluster-node-3 reana.io/system=runtimesessions
+```
+
+#### 3. Upgrade the database schema
+
+This release introduces several database schema changes to support
+among others the new quota accounting feature. We thus need to perform
+database schema upgrade by following the dedicated [ ugprade
```

Comment:
ugprade - > upgrade

Reply:
And extra space in the markdown link: [ upgrade -> [upgrade